crates/pop-telemetry/src/lib.rs

```diff
@@ -0,0 +1,330 @@
+use reqwest::Client;
```

Comment:
Add the license header like in the rest of code we have

Reply:
Out of the scope of this PR imo, but it does need to be addressed.